app/assets/javascripts/components/stories/Sprints.js

```diff
@@ -23,10 +24,10 @@ const renderSprints = (sprints, fetchStories) => {
   );
 };
 
-const Sprints = ({ sprints, fetchStories }) => {
+const Sprints = ({ column, sprints, fetchStories }) => {
   if (!sprints.length) return null;
 
-  return <div className="Sprints">{renderSprints(sprints, fetchStories)}</div>;
+  return <div data-cy={ !column ? 'sprints' : null } className="Sprints">{renderSprints(column, sprints, fetchStories)}</div>;
```

Comment:
If one function is created for get `data-cy` you can use the same in others places too.